benchmark/_cli.js

```diff
@@ -88,12 +88,17 @@ CLI.prototype.benchmarks = function() {
   const filter = this.optional.filter || false;
 
   for (const category of this.items) {
+    if (benchmarks[category] === undefined)
+      continue;
     for (const scripts of benchmarks[category]) {
       if (filter && scripts.lastIndexOf(filter) === -1) continue;
 
       paths.push(path.join(category, scripts));
     }
   }
 
+  if (paths.length === 0)
+    throw new Error('No benchmarks found');
```

Comment:
Maybe I'd add "in the current folder", but I don't run this often enough anyway.